### PR TITLE
Fix external link icons

### PR DESCRIPTION
**Pull request by octogonz**
Eliminate these annoying link icons caused by Docusaurus incorrect detection of external URLs:

rushstack.io
image

api.rushstack.io
image

### CODE REVIEW

File: websites/api.rushstack.io/src/css/custom.css [hunk 1]
```diff
@@ -153,3 +153,19 @@ blockquote {
...
...0%2Fsvg%22 xmlns%3Acc%3D%22http%3A%2F%2Fcreativecommons.org%2Fns%23%22 xmlns%3Adc%3D%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2F%22 xmlns%3Ardf%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22 %3E%3Cmetadata%3E%3Crdf%3ARDF%3E%3Ccc%3AWork%3E%3Cdc%3Acreator%3E%3Ccc%3AAgent%3E%3Cdc%3Atitle%3Ehttps%3A%2F%2Fgithub.com%2Fpgonzal%3C%2Fdc%3Atitle%3E%3C%2Fcc%3AAgent%3E%3C%2Fdc%3Acreator%3E%3C%2Fcc%3AWork%3E%3C%2Frdf%3ARDF%3E%3C%2Fmetadata%3E%3Cg transform%3D%22translate(-82.14 -158.93)%22 fill%3D%22%23bc5835%22 %3E%3Cpath d%3D%22m83.4 161.53c-0.80824-0.0237-1.3558 0.80437-1.2464 1.5543 0.0038 3.237-0.0077 6.4743 0.0058 9.711 0.0539 0.73242 0.77982 1.2548 1.4899 1.151 3.2712-5e-3 6.543 0.01 9.8138-7e-3 0.75077-0.0663 1.2005-0.84784 1.1068-1.5494v-4.0444h-2.1993v3.4019h-8.0176v-8.0176h1.6784v-2.1994h-2.6314z%22%2F%3E%3Cpath d%3D%22m96.504 158.93c-2.5996 0.45395-5.1993 0.90791-7.7989 1.3619 0.56868 0.56868 1.1374 1.1374 1.706 1.706-1.0392 1.0392-2.0785 2.0785-3.1177 3.1177 1.0083 1.0083 2.0166 2.0166 3.025 3.025 1.0392-1.0392 2.0785-2.0785 3.1177-3.1177 0.56868 0.56868 1.1374 1.1374 1.706 1.706 0.45396-2.5996 0.90792-5.1993 1.3619-7.7989z%22%2F%3E%3C%2Fg%3E%3C%2Fsvg%3E');
```
Comment (iclanton) [inline websites/api.rushstack.io/src/css/custom.css:170]: This is kinda nasty. Why not just check in the SVG file and use style-loader's handling of `url` references?

File: websites/rushjs.io/src/css/custom.css [hunk 2]
```diff
@@ -153,3 +153,19 @@ blockquote {
...
...0%2Fsvg%22 xmlns%3Acc%3D%22http%3A%2F%2Fcreativecommons.org%2Fns%23%22 xmlns%3Adc%3D%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2F%22 xmlns%3Ardf%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22 %3E%3Cmetadata%3E%3Crdf%3ARDF%3E%3Ccc%3AWork%3E%3Cdc%3Acreator%3E%3Ccc%3AAgent%3E%3Cdc%3Atitle%3Ehttps%3A%2F%2Fgithub.com%2Fpgonzal%3C%2Fdc%3Atitle%3E%3C%2Fcc%3AAgent%3E%3C%2Fdc%3Acreator%3E%3C%2Fcc%3AWork%3E%3C%2Frdf%3ARDF%3E%3C%2Fmetadata%3E%3Cg transform%3D%22translate(-82.14 -158.93)%22 fill%3D%22%23bc5835%22 %3E%3Cpath d%3D%22m83.4 161.53c-0.80824-0.0237-1.3558 0.80437-1.2464 1.5543 0.0038 3.237-0.0077 6.4743 0.0058 9.711 0.0539 0.73242 0.77982 1.2548 1.4899 1.151 3.2712-5e-3 6.543 0.01 9.8138-7e-3 0.75077-0.0663 1.2005-0.84784 1.1068-1.5494v-4.0444h-2.1993v3.4019h-8.0176v-8.0176h1.6784v-2.1994h-2.6314z%22%2F%3E%3Cpath d%3D%22m96.504 158.93c-2.5996 0.45395-5.1993 0.90791-7.7989 1.3619 0.56868 0.56868 1.1374 1.1374 1.706 1.706-1.0392 1.0392-2.0785 2.0785-3.1177 3.1177 1.0083 1.0083 2.0166 2.0166 3.025 3.025 1.0392-1.0392 2.0785-2.0785 3.1177-3.1177 0.56868 0.56868 1.1374 1.1374 1.706 1.706 0.45396-2.5996 0.90792-5.1993 1.3619-7.7989z%22%2F%3E%3C%2Fg%3E%3C%2Fsvg%3E');
```
Comment (iclanton) [inline websites/rushjs.io/src/css/custom.css:170]: Same here.

File: websites/rushstack.io/src/css/custom.css [hunk 3]
```diff
@@ -153,3 +153,19 @@ blockquote {
...
...0%2Fsvg%22 xmlns%3Acc%3D%22http%3A%2F%2Fcreativecommons.org%2Fns%23%22 xmlns%3Adc%3D%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2F%22 xmlns%3Ardf%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22 %3E%3Cmetadata%3E%3Crdf%3ARDF%3E%3Ccc%3AWork%3E%3Cdc%3Acreator%3E%3Ccc%3AAgent%3E%3Cdc%3Atitle%3Ehttps%3A%2F%2Fgithub.com%2Fpgonzal%3C%2Fdc%3Atitle%3E%3C%2Fcc%3AAgent%3E%3C%2Fdc%3Acreator%3E%3C%2Fcc%3AWork%3E%3C%2Frdf%3ARDF%3E%3C%2Fmetadata%3E%3Cg transform%3D%22translate(-82.14 -158.93)%22 fill%3D%22%23bc5835%22 %3E%3Cpath d%3D%22m83.4 161.53c-0.80824-0.0237-1.3558 0.80437-1.2464 1.5543 0.0038 3.237-0.0077 6.4743 0.0058 9.711 0.0539 0.73242 0.77982 1.2548 1.4899 1.151 3.2712-5e-3 6.543 0.01 9.8138-7e-3 0.75077-0.0663 1.2005-0.84784 1.1068-1.5494v-4.0444h-2.1993v3.4019h-8.0176v-8.0176h1.6784v-2.1994h-2.6314z%22%2F%3E%3Cpath d%3D%22m96.504 158.93c-2.5996 0.45395-5.1993 0.90791-7.7989 1.3619 0.56868 0.56868 1.1374 1.1374 1.706 1.706-1.0392 1.0392-2.0785 2.0785-3.1177 3.1177 1.0083 1.0083 2.0166 2.0166 3.025 3.025 1.0392-1.0392 2.0785-2.0785 3.1177-3.1177 0.56868 0.56868 1.1374 1.1374 1.706 1.706 0.45396-2.5996 0.90792-5.1993 1.3619-7.7989z%22%2F%3E%3C%2Fg%3E%3C%2Fsvg%3E');
```
Comment (iclanton) [inline websites/rushstack.io/src/css/custom.css:170]: And here.

**Review by iclanton — APPROVED** [review]

File: websites/api.rushstack.io/src/css/custom.css (hunk 1 above)
Comment (octogonz) [reply websites/api.rushstack.io/src/css/custom.css:170]: This CSS fragment was blindly copy+pasted from the old Jekyll project.

I'll give that a try.  However the Webpack config is internal to Docusaurus, so we'll have to see whether they support it.

**Pull request merged**